contracts/BaseEscrowFactory.sol

```diff
@@ -61,12 +73,35 @@ abstract contract BaseEscrowFactory is IEscrowFactory, ResolverValidationExtensi
         uint256 takingAmount,
         uint256 remainingMakingAmount,
         bytes calldata extraData
-    ) internal override(ResolverValidationExtension) {
+    ) internal override(FeeTaker) {
+        address integratorFeeRecipient = address(bytes20(extraData[:20]));
+        address protocolFeeRecipient = address(bytes20(extraData[20:40]));
+
+        extraData = extraData[40:];
+
         uint256 superArgsLength = extraData.length - SRC_IMMUTABLES_LENGTH;
-        super._postInteraction(
-            order, extension, orderHash, taker, makingAmount, takingAmount, remainingMakingAmount, extraData[:superArgsLength]
+
+        (uint256 integratorFeeAmount, uint256 protocolFeeAmount, bytes calldata tail) = FeeTaker._getFeeAmounts(
+            order, 
+            taker, 
+            takingAmount, 
+            makingAmount, 
+            extraData[:superArgsLength]
         );
 
+        if (tail.length > 19) {
+            IPostInteraction(address(bytes20(tail))).postInteraction(
```

Comment:
I see the value of the vertical format for quick arg-by-arg reading, but in this case we're just forwarding unchanged parameters. Putting the call on one line would align with the style used in our other repos. No strong push here - just a consistency suggestion, feel free to keep it multi-line if you prefer.